README.md

```diff
@@ -71,13 +71,13 @@ TO DRIVES THAT ARE VENDOR SAMPLES OR NOT PUBLICLY AVAILABLE.
 
 ## II. Compilation and installation
 
-### II.1. Requiremensts
+### II.1. Requirements
 
-libzbc requires  that the  autoconf, automake and  libtool development
+libzbc requires that the autoconf, automake and libtool development
```

Comment:
The extra spaces are there to align line length... Keep them please.

Reply:
that's pretty much anti-markdown @damien-lemoal

Reply:
Yes and no. Rendering of the file in a web browser will ignore the spaces, but readability is in my opinion better when the file is viewed in a terminal.

Reply:
Resolved